plugins/ips/ips.plugin.zsh

```diff
@@ -0,0 +1,3 @@
+# easy get ip - internal & external
+alias ipe="curl ipinfo.io/ip"
+alias ipi="ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*'| grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1'"
```

Comment:
What about merging the aliases ?
```suggestion
alias ip="curl -w '\n' ipinfo.io/ip && ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*'| grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1'"
```
Or adding a new one
```suggestion
alias ipe="curl ipinfo.io/ip"
alias ipi="ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*'| grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1'"
alias ip="ipe && ipi"
```

Reply:
thanks  @cmalard but "ip" its a command in Linux, that why I avoided use it